### PR TITLE
Rename parameter 'cancellationtoken', Remove unread field '_mapper' a…

### PR DESCRIPTION
**Pull request by OlhaPivniuk**
…nd _repository, Rename parameter 'extension' to 'mimeType', Rename parameter 'erroMsg'

dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.BLL/Services/BlobStorageService/BlobService.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public string FindFileInStorageAsBase64(string name)
         return base64;
     }
 
-    public string SaveFileInStorage(string base64, string name, string extension)
+    public string SaveFileInStorage(string base64, string name, string mimeType)
     {
```
Comment (yaninavoda) [inline Streetcode/Streetcode.BLL/Services/BlobStorageService/BlobService.cs:47]: I don't get it. Please explain why mimetype, not extension?

Comment (OlhaPivniuk) [reply Streetcode/Streetcode.BLL/Services/BlobStorageService/BlobService.cs:47]: to match the interface declaration. Parameter names should match base declaration - it is Sonarcloud recommendation
image

**Review by HlibBondarev — APPROVED** [review]
Ok

File: Streetcode/Streetcode.BLL/Services/BlobStorageService/BlobService.cs (hunk 1 above)
Comment (yaninavoda) [reply Streetcode/Streetcode.BLL/Services/BlobStorageService/BlobService.cs:47]: I might be mistaking but in my opinion the mistake is in IBlobService interface. The replacement of mimeType TO extension should be made there.

Comment (OlhaPivniuk) [reply Streetcode/Streetcode.BLL/Services/BlobStorageService/BlobService.cs]: Resolved

**Review by yaninavoda — APPROVED** [review]

**Pull request merged**